### PR TITLE
update about page

### PR DESCRIPTION
**Pull request by mymindstorm**

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -59,6 +59,12 @@ fi
 ./node_modules/sass/sass.js sass:css
 cp ./sass/DroidSansMono.woff2 ./css/
 
+echo "Generating licenses file..."
+./node_modules/.bin/npm-license-generator \
+    --out-path ./view/licenses.html \
+    --template ./scripts/licenses-template.html \
+    --error-missing=true
```
Comment (Sneezry) [inline scripts/build.sh:66]: <del>`--registry` should be a requied parameter right?</del>

Comment (Sneezry) [reply scripts/build.sh:66]: also, I get `URL` is not defined error from `Authenticator/node_modules/npm-license-generator/build/index.js:100:21`

**Review by Sneezry — CHANGES_REQUESTED** [review]

File: scripts/build.sh (hunk 1 above)
Comment (mymindstorm) [reply scripts/build.sh:66]: registry is optional and defaults to registry.npmjs.com. What version of node are you using?

Comment (Sneezry) [reply scripts/build.sh:66]: It's Node 8. I'm upgrading to 12 to test again.

Comment (mymindstorm) [reply scripts/build.sh:66]: URL was considered experimental in node 8, I need to add an import for it.

Comment (Sneezry) [reply scripts/build.sh:66]: I have verified on Node 12, no errors in build.

**Review by Sneezry — APPROVED** [review]

File: scripts/build.sh (hunk 1 above)
Comment (mymindstorm) [reply scripts/build.sh:66]: node 8 is supported with 1.0.1

**Pull request merged**